libtsuba/include/tsuba/RDG.h

```diff
@@ -107,7 +107,7 @@ class KATANA_EXPORT RDG {
   void set_rdg_dir(const katana::Uri& rdg_dir) { rdg_dir_ = rdg_dir; }
 
   uint32_t partition_number() const { return partition_number_; }
-  void set_partition_number(uint32_t partition_number) {
+  void set_partition_id(uint32_t partition_number) {
```

Comment:
would prefer this matches the member name, in this case: `set_partition_number`. If `id` is better than `number` here fix the accessor as well please (and probably the field name too).